### DISCUSSION TITLE
[07/11/2024] Score Community Meeting

### DISCUSSION THREAD

**Discussion by sujaya-sys** [score-spec/spec#107] [Community meetings]
**Info**

| Community call | Info |
|:-----------|:------------|
| Meeting Link | Join via [Google Meet](https://meet.google.com/znt-usdc-hzs) or call +49 40 8081618260 (Pin: 599 887 196)
| Meeting Agenda & Notes | Add to our agenda or review minutes [here](https://github.com/score-spec/spec/discussions/categories/community-meetings)
| Meeting Time | 1:00-2:00pm UTC, every first Thursday of the month

**Agenda**

* Number of releases: 5
  * `score-go`: 3
  * `score-compose`: 3
  * `score-k8s`: 4
* New contributors, thank you!
  * https://score.devstats.cncf.io/d/52/new-contributors-table?orgId=1
  * Want to contribute? --> https://clotributor.dev/search?foundation=cncf&project=score
* Docs updates:
  *  https://docs.score.dev/docs/score-implementation/
* Blogs:
  * https://score.dev/blog/score-at-kubecon-na-in-slc/
* Misc:
  * https://github.com/score-spec/score-implementation-sample
  * https://github.com/score-spec/community-provisioners
* Review of [open PRs](https://github.com/pulls?q=is%3Aopen+is%3Apr+archived%3Afalse+user%3Ascore-spec) and  newly added [issues](https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+user%3Ascore-spec+)

**To Do's in Progress**

- [x] Project Plan for KubeCon NA 2024 SLC: https://github.com/orgs/score-spec/projects/3, let's complete as much as possible before Nov 11th 2024 please ;)
- [ ] Let's file issue(s) to support a new `score-* resources available-provisioners` command generating in the output this table or a more advanced/detailed version with `-o json`: https://docs.score.dev/docs/score-implementation/score-compose/resources-provisioners/? In `score-go` with that it's common across all the implementations?
- [ ] Let's add a new "Examples" entry in the docs where we port all of the examples from there https://github.com/score-spec/score-compose/tree/main/examples?
  - 1 page per Score file?
  - Where to store the Score files (because common to `score-compose` and `score-k8s`)? In a new dedicated `examples` repo?
  - Where do we show the `score-compose init|generate`+`compose up` or `score-k8s init|generate`+`kubectl apply -f`? Under this section https://docs.score.dev/docs/score-implementation/? Or on each example page with 2 tabs (`score-compose` and `score-k8s`)?
- [x] No progress for the score-azure implementation (needs more refinements, focusing on Azure Container Apps)
  - [ ] Starting from here: https://github.com/score-spec/score-implementation-sample/tree/main
  - [ ] Ben/Susa to connect with Andrej
- [ ] Create issue to turn files into key value pair rather than array @chrishumanitec

**Comment by astromechza** [comment]
Can we chat about how we can quantify or drive Score adoption now that we've got more colaboration interest?

**Reply by mathieu-benoit** [reply]
- GHA action usage
- CLIs usage
- Website and Docs traffic

**Comment by sujaya-sys** [comment]
Added a draft for a new blog post: https://docs.google.com/document/d/1uLXQJhyiVsFVcyx6fYX6jbQGgbznE7geSji3wVyHSoM/edit?tab=t.0

**Comment by astromechza** [comment]
Apologies, won't be able to make this session as I'm off sick, I'll catch up with @mathieu-benoit tomorrow /another day

**Comment by mathieu-benoit** [comment]
Discussion around how we can move forward with https://github.com/score-spec/spec/issues, that are change the spec, how to get quorum:
- Design/Suggestion review announcement
- Period to ask for feedback
- Approval by X people at minimum (maintainers votes)
- Documenting details of concrete implementation
- Official announcement about decisions
- Ask for contributors

Needs to update the governance doc?

One concrete one to start with next could be: https://github.com/score-spec/spec/issues/104.

**Comment by sujaya-sys** [comment]
Everything that we didn't get to discussing has been moved to the next community call: https://github.com/score-spec/spec/discussions/109
